src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,29 +1,81 @@
 # TO-DO: Complete the selection_sort() function below 
+
+
+
+
+
+
+
+
+
+
+
+
+def swap(a, b):
```

Comment:
I like the thought behind creating a function to do this, but the change we discussed in 1:1 will work better for this situation.